src/System.IO.FileSystem/src/System/IO/Win32FileSystemObject.cs

```diff
@@ -64,7 +64,7 @@ public DateTimeOffset CreationTime
                 {
                     EnsureDataInitialized();
                     long dt = ((long)(_data.ftCreationTimeHigh) << 32) | ((long)_data.ftCreationTimeLow);
-                    return DateTimeOffset.FromFileTime(dt);
+                    return DateTimeOffset.FromFileTime(dt).ToLocalTime();
```

Comment:
This appears to be an inconsistency between DateTimeOffset.FromFileTime and DateTimeOffset.FromUnixTimeSeconds.  FromFileTime does "ToLocalTime()" inside of it.  But FromUnixTimeSeconds leaves the DateTime as Unspecified.  This inconsistency might be the root cause of this bug, since the Win32FileSystemObject was just calling FromFileTime - when the UnixFileSystemObject was written, it just called FromUnixTimeSeconds without doing the conversion.

It feels like these two methods should be consistent in their output - both should return the same DateTimeKind.